tests/php/Integration/TestTemplateOutput.php

```diff
@@ -27,8 +27,8 @@ class TestTemplateOutput extends AbstractAttribute {
 	 *
 	 * @inheritdoc
 	 */
-	public function setUp() {
-		parent::setUp();
+	public function set_up() {
```

Comment:
Changing `setUp()` to `set_up()`, like [in Core](https://github.com/WordPress/wordpress-develop/blob/6eeaa587e780e294d5b33c75299fc6909a7b63b5/tests/phpunit/tests/attachment/slashes.php#L15), allows this to not have a `:void` return declaration, without causing a PHP error.

Reply:
It also prevents having to have an ugly `phpcs:ignore` on this line.